specification/sbi_cove.adoc

```diff
@@ -401,6 +425,41 @@ image::tvm_destruction_and_memory_reclamation.svg[]
 [title= "TVM runtime execution"]
 image::tvm_runtime_execution.svg[]
 
+
+
+== COVE Enumeration Extension (EID #0x434F5645 "COVE")
+
+This common CoVE extension enumerates capabilities for CoVE such as number of
+active supervisor domains and capabilities of each CoVE supervisor domain.
+
+[#sbi_cove_get_active_domains]
+=== Function: Enumerate active supervisor domains that support CoVE (FID #0)
+[source, C]
+-----
+struct sbiret sbi_cove_get_active_domains(unsigned long active_domains);
+
+-----
+
+Returns a 64 bit vector with bits set for supervisor domains that are active.
+Default value is 1 since supervisor domain 0 is always required (the hosting
+domain). For each non-0 position bit set, the SDID with the value of that bit
+position may be used per the <<cove-fid>> convention to invoke COVH/COVI
+functions. For active domains, the `sbi_covh_get_tsm_info` must be invoked to
+get information about supported TSM capabilities.
```

Comment:
I think it would be good to describe the responsibilities/expectation when supervisor domain switch happens for a running VM. Probably not here but as a broader section in the spec.

Reply:
lines 17-20 do describe the responsibilities of the host and RDSM expected when a function is invoked for/from a specific supervisor domain or workload within a supervisor domain.